content/blog/2022-01-15-moving-local-experiments-to-the-cloud-with-iterative-terraform-provider.md

```diff
@@ -196,47 +197,57 @@ Take a look at the script. In the simplest scenario, all we need to do on a new
 machine to run the training script is to set up the Python environment with
 required libraries. If you simply want to train
 your model on a machine with more memory, this may be enough. However, if you
-want your training code to leverage GPUs, the script will look a bit more
-complex.
+want your training code to leverage GPUs, the script will look a bit different.
 
 ## Training with GPU
 
-Setting up the environment to leverage GPU devices is a bit involved as you need
-to make sure you have the required drivers, NVIDIA-docker, and CUDA.
+There are several ways you can leverage GPU devices on a remote machine. You can install all 
+the required drivers and dependencies "manually" via a script, you can use an existing Docker image 
+or build your own, or, in the case of AWS, you can take advantage of an existing pre-configured Deep Learning AMI. 
```

Comment:
probably should link to choices in docs

```suggestion
or build your own, or, in the case of AWS, you can take advantage of an
[existing pre-configured Deep Learning AMI](https://docs.aws.amazon.com/dlami/latest/devguide/options.html). 
```

Reply:
Fixed in f866dac126e9303660ce02c1934a4bc215360d49